crt_portal/cts_forms/templates/forms/complaint_view/show/actions.html

```diff
@@ -92,6 +92,16 @@
       </button>
     </div>
   </fieldset>
+  <script nonce="{{request.csp_nonce}}">
+    function resetChanges() {
+      const form = document.getElementById('complaint-view-actions');
+      form.reset();
+      [...form.getElementsByClassName('usa-combo-box')].forEach(combobox => {
+        const select = combobox.getElementsByTagName('select')[0];
+        select.nextSibling.value = select.value;
+      });
+    }
+  </script>
```

Comment:
this code is great - straightforward and extensible and covers more than just the DJ number combobox 👏